unreleased_history/behavior_changes/sequence_file_reader_io_pri.md

```diff
@@ -0,0 +1 @@
+SequenceFileReader now sets and passes `IOOptions:rate_limiter_priority` to underlying file system for reads
```

Comment:
Release notes should ideally tell the user what to expect as different or new. SequenceFileReader is internal.